suites/quincy/rgw/tier-1_ssl_rgw_ms_ibm_upgrade_5-3_latest_to_6x_latest.yaml

```diff
@@ -0,0 +1,452 @@
+# Test suite for evaluating RGW multi-site deployment scenario.
+#
+# This suite deploys a single realm (India) spanning across two RHCS clusters. It has a
+# zonegroup (shared) which also spans across the clusters. There exists a master (pri)
+# and secondary (sec) zones within this group. The master zone is part of the pri
+# cluster whereas the sec zone is part of the sec datacenter (cluster).
+
+# The deployment is evaluated by running IOs across the environments.
+# conf : conf/quincy/rgw/rgw_multisite.yaml
+tests:
+
+  - test:
+      abort-on-fail: true
+      desc: Install software pre-requisites for cluster deployment.
+      module: install_prereq.py
+      name: install vm pre-requsites
+
+  - test:
+      abort-on-fail: true
+      clusters:
+        ceph-pri:
+          config:
+            verify_cluster_health: true
+            steps:
+              - config:
+                  command: bootstrap
+                  service: cephadm
+                  args:
+                    custom_image: "cp.stg.icr.io/cp/ibm-ceph/ceph-5-rhel8:5-4"
```

Comment:
HI @anrao19 

For IBM 5.x we can use latest tag (GA)
custom_image: "cp.stg.icr.io/cp/ibm-ceph/ceph-5-rhel8:latest"
custom_repo: "https://public.dhe.ibm.com/ibmdl/export/pub/storage/ceph/ibm-storage-ceph-5-rhel-9.repo"

Reply:
made changes